added_mass/proposal.md

```diff
@@ -0,0 +1,265 @@
+
+$$
+    M
+    =
+    \begin{bmatrix}
+      m           & 0           & 0           & 0           &  m z\_{CoM} & -m y\_{CoM} \\\
+      0           & m           & 0           & -m z\_{CoM} & 0           &  m x\_{CoM} \\\
+      0           & 0           & m           &  m y\_{CoM} & -m x\_{CoM} & 0           \\\
+      0           & -m z\_{CoM} &  m y\_{CoM} & I\_{xx}     & I\_{xy}     & I\_{xz}     \\\
+       m z\_{CoM} & 0           & -m x\_{CoM} & I\_{xy}     & I\_{yy}     & I\_{yz}     \\\
+      -m y\_{CoM} &  mx\_{CoM}  & 0           & I\_{xz}     & I\_{yz}     & I\_{zz}
+    \end{bmatrix}
+$$
+
+where
+
+* \\(m\\) : Body's mass
+* \\(I\_{xx}\\) : Principal mass moment of inertia about the X axis
+* \\(I\_{xy}\\) : Product mass moment of inertia about the X and Y axes, and vice-versa
+* \\(I\_{xz}\\) : Product mass moment of inertia about the X and Z axes, and vice-versa
+* \\(I\_{yy}\\) : Principal mass moment of inertia about the Y axis
+* \\(I\_{yz}\\) : Product mass moment of inertia about the Y and Z axes, and vice-versa
+* \\(I\_{zz}\\) : Principal mass moment of inertia about the Z axis
+* \\(x\_{CoM}\\) : Center of mass X coordinate
+* \\(y\_{CoM}\\) : Center of mass Y coordinate
+* \\(z\_{CoM}\\) : Center of mass Z coordinate
+
+\\(\bf{M}\\) is sufficient to represent the body's inertia when the density of the
+surrounding fluid is much less than the density of the body, and this matrix is
+formed by sdf descriptions of \\(m\\), \\((x\_{CoM},y\_{CoM},z\_{CoM})\\), and the
+moment of inertial matrix \\(\bf I\\), and so no access is given to the individiual
+components of \\(\bf M\\).
+
+When the density of the surrounding fluid is not negligible compared to the
+density of the body, the added mass matrix \\(\bf{\mu}\\) must be included for
+accurate simulations.  This situation commonly arises for submerged and floating
+bodies which have average densities comparable to the density of water, or very
+lightweight objects such as ballons in air.  In this case, \\(\bf{\mu}\\) is
+symmetric and contains 21 unique values in the most general case. The
+off-diagonal terms result physically from the situation in which acceleration of
+the body in one direction results in an acceleration of the surrounding fluid in
+a different direction.  [2]
+
+$$
+    \bf{\mu}
+    =
+    \begin{bmatrix}
+      xx & xy & xz & xp & xq & xr \\\
+      xy & yy & yz & yp & yq & yr \\\
+      xz & yz & zz & zp & zq & zr \\\
+      xp & yp & zp & pp & pq & pr \\\
+      xq & yq & zq & pq & qq & qr \\\
+      xr & yr & zr & pr & qr & rr
+    \end{bmatrix}
+$$
+
+where
+
+* \\(xx\\) : added mass in the X axis due to linear acceleration in the X axis
+* \\(xy\\) : added mass in the X axis due to linear acceleration in the Y axis, and vice-versa
+* \\(xz\\) : added mass in the X axis due to linear acceleration in the Z axis, and vice-versa
+* \\(xp\\) : added mass in the X axis due to angular acceleration about the X axis, and vice-versa
```

Comment:
Would it be worth specifying units?

Reply:
I added the units, but I'd like to confirm with @hamilton8415 or @andermi if they're correct.